pod/perlfunc.pod

```diff
@@ -9260,6 +9262,42 @@ The transliteration operator.  Same as
 L<C<yE<sol>E<sol>E<sol>>|/yE<sol>E<sol>E<sol>>.  See
 L<perlop/"Quote-Like Operators">.
 
+=item trim EXPR
+X<trim> X<remove_whitespace>
+
+=item trim
+
+=for Pod::Functions strip whitespace from the beginning and end of a string
+
+This function returns a string with whitespace stripped from the beginning
+and end of string. trim() will remove these characters:
+
+" ", an ordinary space.
+
+"\t", a tab.
+
+"\n", a new line (line feed).
+
+"\r", a carriage return.
+
+and other unicode characters that are flagged as whitespace.
```

Comment:
Is it possible to provide a link to what counts as “whitespace” as per Unicode?

For example, do VT or FF characters?

If it’s not possible to reference Unicode, it would at least be helpful, IMO, to name all of the ASCII characters that Unicode considers “whitespace”. That’d be redundant, but no less so than listing \t.

Reply:
It is the same as the regex `\s` character class under unicode rules, which links to https://perldoc.perl.org/perlunicode#Unicode-Character-Properties as an explanation.

Reply:
I'm not sure it's particularly helpful though.

Reply:
https://perldoc.perl.org/perlrecharclass#Whitespace is a good explanation.

Reply:
Actually—now that I reread this—the s/// equivalent below should be clear enough … though I do think it would make the reader’s life a bit simpler if a quick link were provided that listed the relevant characters.

Reply:
Unicode should be capitalized.